core/src/main/java/org/kiwix/kiwixmobile/core/search/SearchActivity.kt

```diff
@@ -110,8 +111,10 @@ class SearchActivity : BaseActivity() {
     val searchMenuItem = menu.findItem(id.menu_search)
     searchMenuItem.expandActionView()
     searchView = searchMenuItem.actionView as SearchView
-    searchView.setOnQueryTextListener(SimpleTextListener {
+    searchView.setOnQueryTextListener(SimpleTextListener({
```

Comment:
same here

Reply:
ah actually not same here, when the parameters of a function is a series of lambdas I generally prefer not to put any outside the `()`s to maintain a consistent look.

Reply:
Removing the search button related stuff ended up making this change unnecessary, therefore it is removed now